lib/runtime-core/src/backend.rs

```diff
@@ -143,6 +143,11 @@ pub struct CompilerConfig {
     /// These verifications are disabled by default in 'release' builds.
     pub enable_verification: bool,
 
+    /// Should we automatically attach the custom sections to the Module?
+    ///
+    /// It's off by default as it takes more cpu and memory to read and store the sections.
```

Comment:
Well, I would argue that it should be `on` by default.
We should offer everything to the user by default, and then it's up to the user to disable things according to its context and constraints.

Reply:
I agree, plus that way is a simpler change.  Syrus and I discussed this yesterday and it seems for now we're going to have that overhead but that an ideal design removes the overhead even when we parse custom sections.  All we really need to do if we keep the Wasm module in memory, is store offsets into it for the custom sections, requiring no copying or real overhead (not even a double parse, as we do now, if our planned improvement of our parsing logic goes through)